crt_portal/cts_forms/templates/forms/complaint_view/dashboard/filters/assignee.html

```diff
@@ -0,0 +1,14 @@
+{% extends "forms/complaint_view/dashboard/filter.html" %}
+
+{% block controls %}assigned_to{% endblock %}
+{% block label %}Intake Specialist{% endblock %}
+{% block id %}assigned_to{% endblock %}
+{% block content %}
+   <label for="id_assigned_to">
+    <span class="usa-sr-only">assignee</span>
+   </label>
+   <label for="id_assigned_to-select">
+    <span class="usa-sr-only">assignee</span>
+   </label>
```

Comment:
Does this label do anything? I've removed it from the other branch where I was working on the assignee filter, because the label `for` attribute doesn't appear to target any element ID on the page, and it has the exact same content as the label above it.